File: Level_1/두 개 뽑아서 더하기/Sum.java
import java.util.ArrayList;
import java.util.Arrays;
class Solution {
    public int[] solution(int[] numbers) {
        int len = numbers.length;
        int tmp;        
        ArrayList<Integer> list = new ArrayList<Integer>();
        for(int i = 0; i < len; i++){
            for(int j = i+1; j < len; j++){
                tmp = numbers[i] + numbers[j];
                if(!list.contains(tmp)){
                    list.add(tmp);
                }   
            }
        }
        int[] answer = new int[list.size()];
        for(int i = 0; i < list.size(); i++)
            answer[i] = list.get(i);
        Arrays.sort(answer);
        
        return answer;
    }
}

File: Level_1/두 정수 사이의 합/sum.js
function solution(a, b) {
    let answer = 0;
    
    if(a > b){
        let tmp = b;
        b = a;
        a = tmp;
    }
    
    for(let i = a; i <= b; i++)
        answer += i;
    
    return answer;
}
